ext/POSIX/t/wrappers.t

```diff
@@ -164,7 +164,7 @@ SKIP: {
 
 is(-e NOT_HERE, undef, NOT_HERE . ' does not exist');
 
-foreach ([undef, 0, 'chdir', NOT_HERE],
+foreach ([undef, $^V ge v5.41.1 ? !1 : 0, 'chdir', NOT_HERE],
```

Comment:
POSIX isn't dual-life so this shouldn't be necessary

Reply:
Good point. I've removed the version check.